pkg/chartutil/coalesce.go

```diff
@@ -285,9 +286,49 @@ func coalesceTablesFullKey(printf printFn, dst, src map[string]interface{}, pref
 			} else {
 				printf("warning: cannot overwrite table with non table for %s (%v)", fullkey, val)
 			}
+		} else if islist(val) {
+			if islist(dv) {
+				dst[key] = coalesceListsFullKey(printf, dv.([]interface{}), val.([]interface{}), fullkey, merge)
+			} else {
+				printf("warning: cannot overwrite list with non list for %s (%v)", fullkey, val)
+			}
 		} else if istable(dv) && val != nil {
 			printf("warning: destination for %s is a table. Ignoring non-table value (%v)", fullkey, val)
 		}
 	}
 	return dst
 }
+
+// coalesceListsFullKey merges a source list into a destination list.
+//
+// dest is considered authoritative.
+func coalesceListsFullKey(printf printFn, dst, src []interface{}, prefix string, merge bool) []interface{} {
```

Comment:
Might be worth updating the tests in `coalesce_tests.go` to include a case around the list handling